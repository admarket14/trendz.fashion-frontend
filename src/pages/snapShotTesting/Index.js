import React from 'react';
import style from './Components.module.css';

const Index = () => {
  return (
    <div>
      <h2>Media Queries</h2>
      <p>Resize the browser window.</p>

      <p>Make sure you reach the breakpoint at 800px when resizing this frame.</p>

      <div className={style.left}>
        <p>Left Menu</p>
      </div>

      <div className={style.main}>
        <p>Main Content</p>
      </div>

      <div className={style.right}>
        <p>Right Content</p>
      </div>
    </div>
  );
};

export default Index;
