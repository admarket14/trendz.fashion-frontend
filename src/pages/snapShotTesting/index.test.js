import React from 'react';

import Enzyme from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import toJson from 'enzyme-to-json';

import Index from './Index';

const { mount } = Enzyme;
Enzyme.configure({ adapter: new Adapter() });

describe('<Index/>', () => {
  it('render correctly (snapshot)', () => {
    const tree = mount(<Index />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
