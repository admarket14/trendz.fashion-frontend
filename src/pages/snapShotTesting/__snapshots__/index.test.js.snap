// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index/> render correctly (snapshot) 1`] = `
<Index>
  <div>
    <h2>
      Media Queries
    </h2>
    <p>
      Resize the browser window.
    </p>
    <p>
      Make sure you reach the breakpoint at 800px when resizing this frame.
    </p>
    <div
      className="left"
    >
      <p>
        Left Menu
      </p>
    </div>
    <div
      className="main"
    >
      <p>
        Main Content
      </p>
    </div>
    <div
      className="right"
    >
      <p>
        Right Content
      </p>
    </div>
  </div>
</Index>
`;
